### PR TITLE
Docker setup for MAAV Software

### PR DESCRIPTION
**Pull request by rpash**
This MVP will set up the docker image and has a few commands that let you run the docker container using docker compose. More documentation and usability features will be added soon.

### CODE REVIEW

File: maav.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+CMD=$1
+ARG=$2
+
+# TODO: read this from ~/.maavrc
+MAAV_DIR=/home/rpash/maav
```
Comment (rywunder) [inline maav.sh:7]: There is a TODO here

**Review by rywunder — CHANGES_REQUESTED** [review]

File: maav-init.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```
Comment (rywunder) [inline maav-init.sh:1]: I pulled your repo. This file is not runnable. Please run `chmod +x` on it

Comment (dziedada) [reply maav-init.sh:1]: +1 to Ryan's comment

File: maav.sh (hunk 1 above)
Comment (rpash) [reply maav.sh:7]: This script is not done yet

File: maav-init.sh (hunk 2 above)
Comment (rpash) [reply maav-init.sh:1]: This is done

File: maav.sh (hunk 1 above)
Comment (rywunder) [reply maav.sh:7]: I'm going to resolve this for now so that we can merge the PR. Put this in the backlog

**Review by dziedada — APPROVED** [review]

**Pull request merged**